src/localizations/es.js

```diff
@@ -0,0 +1,128 @@
+import Link from "../link/component";
+import { ReactComponent as LoveEmoji } from "../empty-workspace/love.svg";
+import React from "react";
+
+const pluralRules = new Intl.PluralRules(`es`);
+
+const localization = {
+  dropHint: () => `Coloque los archivos .attheme o .attheme-editor aquí`,
+
+  error_title: () => `Vaya, parece que ha habido un error.`,
+  error_description: () => <React.Fragment>
+    Envie una captura de pantalla del siguiente error a {}
+    <Link href="//t.me/snejugal" isWhite={true}>
+      el desarrollador en Telegram
+    </Link>
+    {} y describe lo que hiciste para que ocurriera el error. Es posible que también deba enviar el archivo del tema oirginal que utilizó.
+  </React.Fragment>,
+  error_dismiss: () => `Para descartar el error, solo tócalo.`,
+
+  emptyWorkspace_title: () => `Comienza a trabajar en tu tema`,
+  emptyWorkspace_createTheme: () => `Crear nuevo tema`,
+  emptyWorkspace_openTheme: () => `Abrir temas existentes`,
+  emptyWorkspace_credits: () => <React.Fragment>
+    Creador por {}
+    <Link href="//t.me/snejugal">@snejugal</Link>
+    {} y {}
+    <Link href="//t.me/AlexStrNik">@AlexStrNik</Link>
+    {} con {}
```

Comment:
Add yourself here, something like `Created by ... and translated by <Link href="//t.me/NeganLucille">@NeganLucille</Link> with <LoveEmoji/>`